app/build.gradle

```diff
@@ -60,8 +60,10 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
 
     //100ms.live SDK
-    implementation 'org.webrtc:google-webrtc:1.0.32006'
-    implementation 'com.github.100mslive:android-sdk:2.0.8'
+    implementation "com.github.100mslive:webrtc:WebRTC-source-stamp-2021-08-20T04_05_25"
+    //implementation 'com.github.100mslive:android-sdk:2.0.8'
```

Comment:
This is fine but you don't need to change either of these if you change jitpack maven to `mavenLocal()` and run `./gradlew build publishToMavenLocal` on the sdk before running the app.

Reply:
okay